cfpb/regulations-site
# Bring in CFPB-specific eRegs assets

This commit brings in eRegs assets specific to CFPB that previously lived in a private "eregsip" repository.

This includes various reg-specific landing pages, the CFPB logo, and some JS to pull down the latest CFPB-specific regs from the Federal Register.

The "default" templates for all regs now include CFPB-specific branding.

All mentions and dependencies on the "eregsip" repository should be gone after this PR (now that #841, self-hosted fonts, has been merged).

To test:

```sh
$ ./frontendbuild.sh
$ EREGS_API_BASE=https://www.consumerfinance.gov/eregs-api/ python manage.py runserver 8001
```

Then visit http://localhost:8001 and browse around the reg landing pages to see the working assets.
